go/stats/export.go

```diff
@@ -42,6 +43,10 @@ import (
 var emitStats = flag.Bool("emit_stats", false, "true iff we should emit stats to push-based monitoring/stats backends")
 var statsEmitPeriod = flag.Duration("stats_emit_period", time.Duration(60*time.Second), "Interval between emitting stats to all registered backends")
 var statsBackend = flag.String("stats_backend", "", "The name of the registered push-based monitoring/stats backend to use")
+var dropDimensions = flag.String("drop_dimension", "", "List of dimensions to be dropped from stats vars")
```

Comment:
The command line flag should include "stats", also given the current implementation, the named dimensions are actually combined into a single value and not really "dropped" from the output.

IMO this should be:

```
var combinedDimensions = flag.String("stats_combined_dimensions", "", "List of dimensions to be combined into a single \"all\" value in exported stats vars")
```